qdrant-landing/content/documentation/cloud/aws-marketplace.md

```diff
@@ -0,0 +1,44 @@
+---
+title: AWS Marketplace
+weight: 60
+---
+
+# Qdrant Cloud on AWS Marketplace
+
+## Overview
+
+Our [AWS Marketplace](https://aws.amazon.com/marketplace/pp/prodview-rtphb42tydtzg) listing makes it simpler to access Qdrant for users who wish to host all of their infrastructure on Amazon Web Services.
```

Comment:
Can we make sure customers understand here that the clusters run on AWS, but within our infrastructure?

Reply:
It is not about hosting, but about *billing*.